src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/ContextMenuLoader.cs

```diff
@@ -68,8 +79,66 @@ public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
             }
 
             return contextMenus;
-        }
-
+        }
+
+        // Function to add the context menu item to run as admin
+        private ContextMenuResult CreateRunAsAdminContextMenu(SearchResult record)
+        {
+            return new ContextMenuResult
+            {
+                PluginName = Assembly.GetExecutingAssembly().GetName().Name,
+                Title = _context.API.GetTranslation("Microsoft_plugin_indexer_run_as_administrator"),
+                Glyph = "\xE7EF",
+                FontFamily = "Segoe MDL2 Assets",
+                AcceleratorKey = Key.Enter,
+                AcceleratorModifiers = (ModifierKeys.Control | ModifierKeys.Shift),
+                Action = _ =>
+                {
+                    var info = new ProcessStartInfo

```

Comment:
Can we please put this code in managedcommon library as this functionality can be shared with other plugins as well?

Reply:
Done. Moved it to a helper class in wox.infrastructure.